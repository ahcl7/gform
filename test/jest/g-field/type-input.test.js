import { generateComponent } from "./common";

describe('GField with type input', function () {
  it('should render correctly', () => {
    const component = generateComponent({ key: 'name', type: 'input', flex: 'md-6', default: 'name' });
    expect(component.html()).toMatchSnapshot();
  });
});
