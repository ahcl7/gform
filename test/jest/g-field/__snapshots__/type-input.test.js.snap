// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GField with type input should render correctly 1`] = `
<g-col-stub class="md-6 px-2">
  <g-text-field-stub modelvalue="test"
                     label="Name"
                     type="text"
  >
  </g-text-field-stub>
</g-col-stub>
`;
